apps/test/unit/templates/teacherDashboard/ParentLetterTest.jsx

```diff
@@ -0,0 +1,50 @@
+import {render} from '@testing-library/react';
+import React from 'react';
+
+import {UnconnectedParentLetter as ParentLetter} from '@cdo/apps/templates/teacherDashboard/ParentLetter';
+import {SectionLoginType} from '@cdo/generated-scripts/sharedConstants';
+import i18n from '@cdo/locale';
+
+describe('ParentLetter', () => {
+  const sampleSection = {
+    id: 7,
+    code: 'ABCDEF',
+  };
+
+  it('Secret words section login type should have secret words instructions', () => {
+    render(
+      <ParentLetter
+        section={{
+          ...sampleSection,
+          loginType: SectionLoginType.word,
+        }}
+        teacherName="Minerva McGonagall"
+      />
+    );
+
+    const parentLetter = document.getElementById('printArea');
+    const loginStep =
+      parentLetter.children[1].children[5].children[1].children[1].textContent;
```

Comment:
Instead of relying on DOM position, would we be able to check for the existence that the text exists on the screen? The test would need to be changed if the DOM changes at all.

```
screen.getByText(i18n.parentLetterSecretWords({secretWords: ''});
```

Otherwise, are we able to add a `data-testid` to the element containing the login step and we use `screen.getByTestId(testid)`?

Reply:
Yes, that makes a lot more sense. I will make this change!